### PR TITLE
Create it po

### PR DESCRIPTION
**Pull request by albanobattistella**

### CODE REVIEW

File: po/it.po [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+msgid "B"
+msgstr "B"
+
+#: src/window.py:137
+msgid "Hexadecimal"
+msgstr "Esadecimale"
+
+#: src/window.py:138
+msgid "RGB"
+msgstr "rgb"
```
Comment (Huluti) [inline po/it.po]: Should be better in uppercase, no?

**Comment by Huluti** [comment]
Thank's for you help!

**Comment by albanobattistella** [comment]
> Thank's for you help!

It's a pleasure!

**Review by Huluti — CHANGES_REQUESTED** [review]

**Comment by albanobattistella** [comment]
..sorry..Fixed!

________________________________
Da: Hugo Posnic <notifications@github.com>
Inviato: venerdì 20 marzo 2020 17:07
A: Huluti/Coulr <Coulr@noreply.github.com>
Cc: albanobattistella <albano_battistella@hotmail.com>; Author <author@noreply.github.com>
Oggetto: Re: [Huluti/Coulr] Create it po (#4)


@Huluti requested changes on this pull request.

________________________________

In po/it.po<https://github.com/Huluti/Coulr/pull/4#discussion_r395734956>:

> +#: src/window.py:101
+msgid "G"
+msgstr "G"
+
+#. Blue label
+#: src/window.py:118
+msgid "B"
+msgstr "B"
+
+#: src/window.py:137
+msgid "Hexadecimal"
+msgstr "Esadecimale"
+
+#: src/window.py:138
+msgid "RGB"
+msgstr "rgb"


Should be better in uppercase, no?

**Comment by Huluti** [comment]
Thank you very much! Very appreciated.

**Pull request merged**